core/src/main/java/org/opensearch/sql/expression/DSL.java

```diff
@@ -118,6 +118,10 @@ public NamedArgumentExpression namedArgument(String argName, Expression value) {
     return new NamedArgumentExpression(argName, value);
   }
 
+  public NamedArgumentExpression namedArgument(String name, String value) {
```

Comment:
which named argument needs this?

Reply:
It's convenience overload that simplifies a very common use case.